src/style/aframe.css

```diff
@@ -35,7 +35,7 @@
 
 .a-canvas.a-grab-cursor:hover {
   cursor: grab;
```

Comment:
as it's the ideal, non-vendor-prefixed one, I would move `cursor: grab;` to be the last property-value style